Classes/Emogrifier.php

```diff
@@ -329,6 +329,7 @@ public function emogrifyBodyContent()
      */
     protected function process(\DOMDocument $xmlDocument)
```

Comment:
This method now needs the "throws" annotation (and an explanation on the same line explaining that the Exception is thrown for invalid XPath).